<ui version="4.0" >
 <class>Kleo::Dialogs::CertifyCertificateDialogPrivate::CertificationOptionsWidget</class>
 <widget class="QWidget" name="Kleo::Dialogs::CertifyCertificateDialogPrivate::CertificationOptionsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>523</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Step 3: &lt;/span>Choose how to certify&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="multipleKeysPage" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Choose which of your keys to sign with:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="keyListView" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="singleKeyPage" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="singleKeyLabel" >
         <property name="text" >
          <string>Certification will be performed using certificate %1.</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item rowspan="2" row="3" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QRadioButton" name="localSignatureRB" >
       <property name="text" >
        <string>Certify only for myself</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="exportableSignatureRB" >
       <property name="text" >
        <string>Certify for everyone to see</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="sendToServerCB" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Send certified certificate to server afterwards</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exportableSignatureRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>sendToServerCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>439</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel" >
     <x>455</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
